src/rebar_app_info.erl

```diff
@@ -478,6 +494,17 @@ is_checkout(#app_info_t{is_checkout=IsCheckout}) ->
 is_checkout(AppInfo=#app_info_t{}, IsCheckout) ->
     AppInfo#app_info_t{is_checkout=IsCheckout}.
 
+%% @doc returns whether the app source exists in the deps dir
+%% and profiles/overrides have been applied to the app info opts
+-spec is_available(t()) -> boolean().
+is_available(#app_info_t{is_available=IsAvailable}) ->
+    IsAvailable.
+
+%% @doc sets whether the app is a available app or not
```

Comment:
should add a bit of doc there to mention that "an app should only be declared available once profiles/overrides have been applied" I guess?